src/frontend/components/DashboardThumbnail/index.spec.tsx

```diff
@@ -246,7 +243,9 @@ describe('<DashboardThumbnail />', () => {
   });
 
   it('creates a new thumbnail and redirects the user to the upload form they click on the replace button', async () => {
-    mockCreateThumbnail.mockResolvedValue(video.thumbnail);
+    fetchMock.mock('/api/thumbnail/', JSON.stringify(video.thumbnail), {
```

Comment:
IMO you should not replace the `createThumbnail` mock by `fetchMock`. Doing mock this way is IMO wrong, you mock the function, not it's implementation. The day you change the `createThumbnail` implementation without changing its behaviour you will also have to update this test and I think it's wrong.